tee-worker/core/parentchain/indirect-calls-executor/src/indirect_calls/litentry/mod.rs

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
 
-pub mod batch_all;
+// pub mod batch_all;
```

Comment:
So I didn't see a need for having a separate `dispatch` for `batch_all` since it will internally execute the calls inside the `batch_all`, So here's how it is handled

This is in `filter_calls.rs` 

```rust
IndirectCall::VerifyIdentity(verify_id, address, hash) => verify_id.dispatch(executor, (address.clone(), *hash, block)),
IndirectCall::BatchAll(calls) => {
	for x in calls.clone() {
	         if let Err(e) = x.dispatch(executor, block) {
		         log::warn!("Failed to execute indirect call in batch all due to: {:?}", e);
			continue
		}
	}
	Ok(())
},
```
```rust
BatchAll(Vec<IndirectCall>),
```
Since all the calls implement the `dispatchable` trait, We can just perform `x.dispatch(executor, block)`